src/content/docs/new-relic-control/agent-control/overview.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Agent Control overview"
+metaDescription: "Agent Control is an agent supervisor that can be used to deploy, update, and manage New Relic agents."
+freshnessValidatedDate: never
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+Agent Control is an agent supervisor that can be used to deploy, update, and manage New Relic agents. 
+
+## Prerequisites
+Before you begin deploying Agent Control to your Kubernetes clusters, ensure that you meet the following prerequisites:
+
+- **Helm 3:** Helm version 3 must be installed on your machine. For installation instructions, see [Installing Helm](https://helm.sh/docs/intro/install).
+- **New Relic license key:** You'll need a New Relic license key to report telemetry to your New Relic account.
+- **Kubernetes cluster name:** Have the name of your Kubernetes cluster ready, as it will be referenced during the installation process.
+- **Auth Domain Manager:** To set up Agent Control and connect it to Fleet Control, you must have the Authentication Domain Manager role.
+- **Existing instrumentation:** If you already have a Kubernetes cluster instrumented with New Relic, you have to uninstall the instrumentation and then re-install it:
+  - Determine the existing instrumentation and configuration within your cluster. This may include New Relic Helm charts and Kubernetes manifests stored in your code repository.
+  - Match your existing instrumentation with the supported agents in [the Agent Control configuration](https://github.com/newrelic/helm-charts/blob/master/charts/agent-control/values.yaml).
+  - Keep existing settings for each agent under the new configuration structure. Refer to [sample configurations](/docs/new-relic-control/agent-control/configuration/#sample-configurations) as needed for formatting examples.
+  - Uninstall your current instrumentation for [in-scope agents](/docs/new-relic-control/getting-started/#scope). Refer to your existing agent documentation for instructions on how to uninstall them, and then prepare your cluster for the new setup.
+  - [Install Agent Control](/docs/new-relic-control/agent-control/setup/#manual) with your selected agents and settings.
+
+<Callout variant="important">
+  Since New Relic Control is in public preview, we recommend using it in lower, non-production environments such as sandbox, development, staging, or testing.
+</Callout>
+
+## Compatibility
+
+The solution is compatible with:
+- Last three minor Kubernetes releases.
+- Minikube, Kind, Amazon Elastic Kubernetes Service (Amazon EKS), Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE).
```

Comment:
@ubanerjeeNR 
We also support GKE autopilot and EKS Auto Mode

Notice that despite the AC supports them, some agents might require specific configs (or be not compatible). I think this is why a few lines below we state:
> For system requirements for the Kubernetes infrastructure agent and the New Relic OpenTelemetry collector, see: [...]

Can you review the wording and add the two new flavours we support?